backend/app/routers/product.py

```
@@ -49,48 +58,42 @@
     return new_product
 
 
-
-@router.put("/products/{product_id}", response_model=schemas.ProductCreate)
+@router.patch("/products/{product_id}", response_model=schemas.ProductCreate)
 def update_product(
     product_id: int,
     updated_product: schemas.ProductCreate,
     db: Session = Depends(get_db),
-    user: User = Depends(get_current_user)
+    user: User = Depends(get_current_user),
```

Ruff (0.8.2)

Unused function argument: `user`

(ARG001)